content/getting-started/sections/why-atom.md

```diff
@@ -3,7 +3,26 @@ title: Why Atom?
 ---
 ### Why Atom?
 
+####  About ATOM  
```

Comment:
Atom should be title-case, not all caps in all instances. See existing headers/references :)